Scripts/Web_Scrappers/Economictimes_Scraper/economictimes_scraper.py

```diff
@@ -0,0 +1,71 @@
+from bs4 import BeautifulSoup
+from lxml import etree
```

Comment:
use defusedxml instead of lxml
![Capture](https://user-images.githubusercontent.com/42397096/95160404-1c01e900-07be-11eb-9f7c-694352d7120a.PNG)